client/settings/express-checkout/index.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+/**
+ * Internal dependencies
+ */
+import ExpressCheckoutsSettings from './express-checkout-settings';
+
+const container = document.getElementById(
+	'wc_stripe-express_checkouts_customizer-container'
```

Comment:
The mixture of dashes and underscores is a bit 😬. Is this a convention in the code?

Reply:
Yeah, that's ugly. Underscores separating words, dashes separating concepts I think makes sense. So this will be changed to `wc_stripe-express_checkouts_customizer_container`